app/graphql/types/invoices/fee_input.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  module Invoices
+    class FeeInput < BaseInputObject
+      description 'Fee input for creating invoice'
+
+      argument :add_on_id, ID, required: true
+      argument :unit_amount_cents, GraphQL::Types::BigInt, required: false
+      argument :units, GraphQL::Types::Float, required: false
+      argument :description, String, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `description` should appear before `units`.